automation/e2e-pipelines/test-files/windows2k22-installer-pipelinerun.yaml

```diff
@@ -0,0 +1,30 @@
+apiVersion: tekton.dev/v1beta1
+kind: PipelineRun
+metadata:
+  generateName: windows2k22-installer-run-
+  labels:
+    pipelinerun: windows2k22-installer-run
+spec:
+  params:
+  - name: winImageDownloadURL
+    value: http://http-server/disk.img
+  - name: preferenceName
+    value: "windows.2k22"
+  - name: autounattendConfigMapName
+    value: windows2k22-autounattend
+  - name: baseDvName
+    value: win2k22
+  - name: isoDVName
+    value: win2k22
+  - name: baseDvNamespace
+    value: kubevirt
+  pipelineRef:
+    name: windows-efi-installer
+  serviceAccountName: pipeline
+  taskRunSpecs:
+    - pipelineTaskName: "modify-windows-iso-file"
+      taskPodTemplate:
+        securityContext:
+          runAsUser: 1001
+          runAsGroup: 1001
+          fsGroup: 1001     
```

Comment:
Why was it necessary again to do this?

Reply:
Modify-windows-iso task requires specific UID  and GID to work properly. This makes sure the task pod has correct IDs